src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/IncrementalBuildTest.cs

```diff
@@ -188,39 +188,35 @@ public void IncrementalCleanDuringClean ()
 		}
 
 		[Test]
-		public void LibraryIncrementalBuild () {
-
-			var testPath = Path.Combine ("temp", TestName);
-			var class1Source = new BuildItem.Source ("Class1.cs") {
-				TextContent = () => @"
-using System;
-namespace Lib
-{
-	public class Class1
-	{
-		public Class1 ()
+		public void LibraryIncrementalBuild ()
 		{
-		}
-	}
-}"
-			};
-			var lib = new XamarinAndroidLibraryProject () {
+			var lib = new XamarinAndroidLibraryProject {
 				ProjectName = "Lib",
-				ProjectGuid = Guid.NewGuid ().ToString (),
 				Sources = {
-					class1Source,
+					new BuildItem.Source ("Class1.cs") {
+						TextContent = () => "public class Class1 { }",
+					}
 				},
 			};
-			using (var b = CreateDllBuilder (Path.Combine (testPath, "Lib"))) {
-				Assert.IsTrue (b.Build (lib), "Build should have succeeded.");
-				Assert.IsTrue (b.LastBuildOutput.ContainsText ("LogicalName=__AndroidLibraryProjects__.zip") ||
+			using (var b = CreateDllBuilder ()) {
```

Comment:
Do we have a test which includes a third level library reference? i.e A->B->C where B and C both have `.aar` files. What are we expecting in that case? the `.aar` files should all end up in the the `A` intermediate path. 
Or are we expecting the users to do A->B->C, A->C to get this to work ?

Reply:
I updated a test to have an App A that reference Lib B, Lib B -> Lib C.

It appears to already work without changing anythinig:

![image](https://user-images.githubusercontent.com/840039/94841322-775b6200-03df-11eb-87aa-b2e758398007.png)

But the test now asserts that the app has an `@(AndroidAsset)` from Lib C without actually referencing it, which is a good thing to test.

Reply:
nice :)